src/chef-mover/apps/chef_objects/test/chef_objects_test_utils.erl

```diff
@@ -84,4 +84,4 @@ read_file(File) ->
 read_file(app, File) -> %% Rebar3
     file:read_file(filename:join([".", "apps", "chef_objects", "test", File]));
 read_file(test, File) -> %% Rebar2
-    file:read_file(filename:join(["..", "test", File])).
\ No newline at end of file
+    file:read_file(filename:join(["..", "test", File])).
```

Comment:
Probably shoulod not edit this file as a part of the change to fix crypto:md5 removal.

Reply:
Don't know how it happened, and I can't even see what the change is in order to revert it.  Will try to checkout older version of this file.

Reply:
Done.